src/americana.js

```diff
@@ -32,7 +32,7 @@ export function buildStyle() {
   return Style.build(
     config.OPENMAPTILES_URL,
     `${baseUrl}/sprites/sprite`,
-    `${baseUrl}/fonts/{fontstack}/{range}.pbf`,
+    "https://osm-americana.github.io/fontstack66/{fontstack}/{range}.pbf",
```

Comment:
This could benefit from having a config override to make it easier to test changes to the fontstack locally.

Reply:
Seems reasonable, just implemented that in https://github.com/ZeLonewolf/openstreetmap-americana/pull/878/commits/0568c400d0a46d6e42101085982b961efbe2406c and https://github.com/ZeLonewolf/openstreetmap-americana/pull/878/commits/e1c519e88731653691ef513f17ad158e5b048af1.